### DISCUSSION TITLE
ERROR: (1230): Invalid element in the configuration: 'alerts_log_path'

### DISCUSSION THREAD

**Discussion by janowo2008** [wazuh/wazuh#25199] [Q&A]
I need to change the location of the archives.log and alerts.log files to a larger capacity drive. 
From Chat-GPT I got the information that "to change the path for archives and alerts, use ln commands or move files manually. There are no dedicated configuration tags in the ossec.conf file to set the log paths".
How I can change this     archives.log and alerts.log  paths configuration   ?

![ERROR_PATH_archives_CHANGE__1](https://github.com/user-attachments/assets/2fd94b44-dc30-4f66-ae54-6b5eed66c786)

![ERROR_PATH_archives_CHANGE__2](https://github.com/user-attachments/assets/d9490212-e499-4133-94f7-22313ce7ad28)

**Comment by MiguelazoDS** [comment]
Hi @janowo2008, 

What you can do is to mount a partition with enough capacity to store your log files in the right directory `/var/ossec/logs `

**Comment by sebasfalcone** [comment]
Inactive thread, please re-open it if necessary
